easybuild/scripts/install-EasyBuild-develop.sh

```diff
@@ -69,6 +69,7 @@ conflict    EasyBuild
 
 prepend-path    PATH            "\$root/easybuild-framework"
 
+prepend-path    PYTHONPATH      "\$root/vsc-base"
```

Comment:
It looks like it should be `prepend-path PYTHONPATH "\$root/vsc-base/lib"` here.